### ISSUE TITLE
False positive for `variable is only used in return`

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#598]
# Bug report

## What's wrong

```python
      def function():
            function_result = function(*args, **kwargs)
            message = '{0} succeed'.format(function.__name__)
            print(colored(message, 'green'))  # noqa: T001
            return function_result
```

Source: https://github.com/wemake-services/kira-setup/blob/master/kira_setup/logging.py#L11

## How is that should be

When variable is separated with an extra statement(s) from `return` it should be treated as a correct one.

## System information

I use `0.9.0`

**Comment by sobolevn** [comment]
On the other hand, how can we confidently say that this is correct?
I would say that since value is not used - it is incorrect. Use `noqa` or `print` the value to fix it.
